### ISSUE TITLE
Debug messages still on

### ISSUE THREAD

**Issue by Commandcracker** [unicornpkg/libunicornpkg#27]
Adding some sort of a debug mode would probably be a good idea 
https://github.com/unicornpkg/libunicornpkg/blob/2ce9515e7d3149623938e23a2877b893ff9e94dc/unicorn/remote.lua#L25-L26

**Comment by tomodachi94** [comment]
Yeah, that's a good idea. Maybe if `CC-Logger` is detected as installed, the program should use that.
